x-pack/plugins/apm/public/components/app/ErrorGroupOverview/index.tsx

```diff
@@ -102,20 +99,27 @@ const ErrorGroupOverview: React.FC = () => {
           <LocalUIFilters {...localUIFiltersConfig} />
         </EuiFlexItem>
         <EuiFlexItem grow={7}>
-          <EuiFlexGroup>
-            <EuiFlexItem>
-              <EuiPanel>
-                <ErrorDistribution
-                  distribution={errorDistributionData}
-                  title={i18n.translate(
-                    'xpack.apm.serviceDetails.metrics.errorOccurrencesChartTitle',
-                    {
-                      defaultMessage: 'Error occurrences',
-                    }
-                  )}
-                />
-              </EuiPanel>
-            </EuiFlexItem>
+          <EuiFlexGroup gutterSize="s">
```

Comment:
I think we need to set a minimum height, as the page loads the panels are jumpy. Are we also seeing a longer query time for the error rate chart versus the error occurrences distribution?

![Kapture 2020-06-04 at 20 53 49](https://user-images.githubusercontent.com/4104278/83799839-b6361100-a6a6-11ea-8fbe-bd34f3cad379.gif)

Reply:
I made some improvements to the ES query. I might be a bit better now.